spec/requests/api/v1/lots_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe "Lots API" do
+  let!(:product) { create(:product) }
+
+  it "can create a lot" do
+    expect { 
+      post "/api/v1/lots", params: {
+        lot: {
+          product_id: Product.first.id
+        }
+      }
+    }.to change {Lot.count}.by 1
+
+    lot = JSON.parse(response.body)
+
+    expect(response.status).to eq 201
+    expect(lot['id']).to be_an Integer
```

Comment:
Could we add something like `expect(lot['product_id']).to eq Product.first.id` ?